### ISSUE TITLE
Basic libp2p tutorial

### ISSUE THREAD

**Issue by enoldev** [protocol/launchpad#180]

**Comment by enolal826** [comment]
@walkerlj0 If you want to take a look at the first libp2p tutorial, it's here https://github.com/protocol/launchpad/pull/252

**Comment by enoldev** [comment]
@walkerlj0 Video ready for review https://drive.google.com/file/d/1NdBSQslK40do5-fjFkiYn631pw4gRdm9/view?usp=sharing

**Comment by walkerlj0** [comment]
Script can be found here under 'resources to review': https://www.notion.so/protocollabs/Launchpad-Tutorials-90553d649d974261ab3cc549dd3242d5

**Comment by walkerlj0** [comment]
@enoldev I have made my changes! If you generally accept, feel free to merge into yours
Only bit of change for the video- add a bit of basic introduction about what this tech _is_ and what you will be doing in more plain language!

Once v2 of video is done, upload to GDrive in the libp2p video folder, add to the spreadsheet, and let me know so I can upload to YouTube!

I'll upload it to youtube, put in the link, and merge to main once you have finalized all changes!

**Comment by enoldev** [comment]
@walkerlj0 new version here https://drive.google.com/file/d/1L30cHeppM4Z1gKbpPPIhKwPSW6gQ3rvU/view?usp=sharing

**Comment by walkerlj0** [comment]
my PR to enols 'PR': https://github.com/protocol/launchpad/pull/286

**Comment by walkerlj0** [comment]
@enoldev I think we're good to go! 
- Commit my PR to you PR: [https://www.notion.so/protocollabs/Script-libp2p-simple-node-4fe315be15b74d849ca25398395a3b6b](https://github.com/protocol/launchpad/pull/286)
- Make any changes you see need changing!
- I updated your script with the correct times from the (final) v2 video: [https://www.notion.so/protocollabs/Script-libp2p-simple-node-4fe315be15b74d849ca25398395a3b6b](https://www.notion.so/protocollabs/Script-libp2p-simple-node-4fe315be15b74d849ca25398395a3b6b)
- Add the description in the [YouTube spreadsheet under libp2p](https://docs.google.com/spreadsheets/d/1uyN_5uL4y6EFQVV7UHpPtiMQ_WWx9nzC36MCtNyCPfY/edit#gid=301049246)

Let me know when that's all done, I'll make the video public. Feel free to merge into main pending no (major) changes
